src/styles/blocks/buy.scss

```diff
@@ -0,0 +1,24 @@
+.buy {
+  &__link {
+    display: block;
+    box-sizing: border-box;
+
+    background-color: #00acdc;
+
+    border-radius: 5px;
+    padding: 12px 70px;
+
+    text-decoration: none;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      color: #00acdc;
+      background-color: #fff;
+      border: 1px solid #00acdc;
```

Comment:
Directly setting a `border` on hover can cause the element's size to change, which may lead to layout shifts. As per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use a transparent border for the normal state and change the border-color on hover to avoid layout shifts.